client/webserver/site/src/js/markets.js

```diff
@@ -692,7 +692,14 @@ export default class MarketsPage extends BasePage {
       if (!app.checkResponse(res)) return
       bttn.parentNode.textContent = 'cancelling'
       order.cancelling = true
-    })
+
+      // If "once" option is not supported by most browsers, we will need to
+      // unbind this function:
+      // Doc.unbind(page.cancelSubmit,'click', handler)
```

Comment:
Looks like `once` is pretty well supported (https://caniuse.com/#feat=mdn-api_eventtarget_addeventlistener_options_once), so the handler can be made back into an arrow function.